button.md

```diff
@@ -0,0 +1,85 @@
+## Quick Start
+
+The Harvest Button works by running a script that handles creating elements and their respective behavior. 
+
+Add Harvest Button placeholders to your application where appropriate:
+
+```html
+<div class="harvest-timer"
+  data-item='{"id":1337,"name":"Write documentation"}'>
+</div>
+```
+
+Then, drop this `<script>` into your application:
+
+```javascript
+<script>
+  (function () {
+    window._harvestPlatformConfig = {
+      "applicationName": "ExampleCompany",
+      "permalink": "https://example.com/item/%ITEM_ID%"
+    };
+    var s = document.createElement("script");
+    s.src = "https://platform.harvestapp.com/assets/platform.js";
+    s.async = true;
+    var ph = document.getElementsByTagName("script")[0];
+    ph.parentNode.insertBefore(s, ph);
+  })();
+</script>
+```
+
+The script will handle it from here! 
+
+If you need to add a Harvest Button after the script has loaded, see [Adding Buttons After Loading](#adding-buttons-after-loaded).
+
+## Configuration Options
+
+These global configuration settings are set on the `window._harvestPlatformConfig` object:
+
+| Parameter         | Description
+|-------------------|-------------
+| `applicationName` | The human-readable name of your application. Example: `Trello`
```

Comment:
Perhaps, for consistency. I’ll see how it looks (the other table I had to use `<pre>` because otherwise it was line-breaking at the hyphen)!